README.adoc

```diff
@@ -51,14 +45,45 @@ IMPORTANT: The core of the Objects server itself is presently proprietary and mu
+
+[width="100%",options="header,footer"]
+|===
+|Service|API links|Configuration|Description
+|Authentication Server|https://api.smartcosmos.net/microservices/smartcosmos-auth-server/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-auth-server.yml[smartcosmos-auth-server.yml]|
+|User Management DevKit|https://api.smartcosmos.net/microservices/smartcosmos-edge-user-devkit/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-edge-user-devkit.yml[smartcosmos-edge-user-devkit.yml]|
+|Things|https://api.smartcosmos.net/microservices/smartcosmos-edge-things/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-edge-things.yml[smartcosmos-edge-things.yml]|
+|Relationships|https://api.smartcosmos.net/microservices/smartcosmos-edge-relationships/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-ext-relationships.yml[smartcosmos-ext-relationships.yml]|
+|Metadata|https://api.smartcosmos.net/microservices/smartcosmos-edge-metadata/index.html[API]|https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-edge-metadata.yml[smartcosmos-edge-metadata.yml]]|
+|Bulk Import|https://api.smartcosmos.net/microservices/smartcosmos-edge-bulkimport/index.html[API]|https://github.çim/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-edge-bulkimport.yml[smartcosmos-edge-bulkimport.yml]|
+Supports the mass import of datasets
+|Gateway||https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-gateway.yml[smartcosmos-gateway.yml]|
+Routing
+|Events||https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-events.yml[smartcosmos-events.yml]|
+|===
+
+Additional services and configuration files:
+
+* Base configuration for all services: https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/application.yml[application.yml]
+* Event listener https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit/blob/master/config/smartcosmos-event-listener.yml[smartcosmos-event-listener.yml]
 
 == SMART COSMOS DevKit
 The DevKit is a composition of several standalone services which
 communicate via REST. Each service runs in its own container.
 To manage these service containers we use https://docker.com[`Docker`].
 
+With the DevKit it is possible to run a PaaS solution that supports a collection of best practices to engage any data processing relevant for your business.
+Aside the composition of several standalone services you will get this:
+
+* A full adaptable cloud-based PaaS
+* A mircoservice architecture centric cluster
+* The TRIM metapher - as best answer to handle IoT
```

Comment:
"TRIM metapher" -> "TRIM metaphor"
BTW: do we already have the 'I' for "Interactions"?